### PR TITLE
Add (my) Todo-Backend implementations

### PR DESCRIPTION
**Pull request by aesteve**
Ceylon impl. is in progress...
The GSOC implementations (Redis, JDBC, Mongo) could/should be added too ! :)

### CODE REVIEW

**Comment by pmlopes** [comment]
the GSoC implementations are added as a single entry in the examples

File: README.md [hunk 1]
```diff
@@ -248,6 +248,10 @@ For Vert.x version 2 check [this page](./vert-x2.md).
...
 * [Vert.x Markdown service](https://github.com/aesteve/vertx-markdown-service) - Example on how to use [service-proxy](https://github.com/vert-x3/vertx-service-proxy) with Gradle.
 * [Example using event bus and service proxies to connect vertx and node](https://github.com/advantageous/vertx-node-ec2-eventbus-example) - Step by step example with wiki description showing how to connect Vert.x and Node using event bus and service proxies.
+* [Vert.x Todo-Backend implementation](https://github.com/aesteve/todo-backend-vertx). Pure Java 8 implementation of the Todo MVC backend. Uses a Vert.x LocalMap for storage.
+* [Kotlin Todo-Backend implementation](https://github.com/aesteve/vertx-kotlin-todomvc). Kotlin implementation of the Todo MVC backend.
+* [Scala Todo-Backend implementation](https://github.com/aesteve/vertx-scala-todomvc) Scala implementation of the Todo MVC backend.
+* [Grooveex Todo-Backend implementation](https://github.com/aesteve/todo-backend-grooveex) : Todo MVC backend implementation with Vert.x + Groovy + some syntaxic sugar + DSL routing facilities
```
Comment (michel-kraemer) [inline README.md]: syntaxic should be syntactic. Please add a dot at the end of the description. Replace the colon ':' by a hyphen '-'. And while you are at it do so for the other entries as well (i.e. hyphen between link and description). Thanks!

**Comment by michel-kraemer** [comment]
@aesteve Thank you so much for contributing these examples! I just have a couple of minor nitpicks.

**Comment by aesteve** [comment]
@michel-kraemer thanks for the help, just tell me if I didn't miss anything.

**Comment by michel-kraemer** [comment]
Thanks, @aesteve. LGTM. Should we merge now or should we wait for the Ceylon example?

**Comment by aesteve** [comment]
Just merge it. Not sure when I'm gonna end the ceylon example. In fact, Ceylon is a very powerful language and I'm pretty sure I could end up with a very very nice implementation by composing functions etc. But for now it's totally messy and wouldn't be a very nice example to show :)

**Pull request merged**